server/conn.go

```diff
@@ -1084,6 +1086,13 @@ func (cc *clientConn) writeChunksWithFetchSize(ctx context.Context, rs ResultSet
 	return errors.Trace(cc.writeEOF(serverStatus))
 }
 
+func (cc *clientConn) chunkCapacity(wantSize int) int {
+	if wantSize > variable.DefMaxChunkSize {
```

Comment:
Ah, I've got a mistake. We should return the current max chunk size here, instead of the default one. And in the next line the return values should not be init chunk size.

Reply:
- - oh fixed.